sklearn/utils/tests/test_arpack.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+from sklearn.utils._arpack import _init_arpack_v0
+
+
+def test_init_arpack_v0():
+    v0s = [_init_arpack_v0(1000, i) for i in range(100)]
+    assert all(-1 <= v <= 1 for v0 in v0s for v in v0)
+
+    v0 = np.concatenate(v0s)
+    assert np.allclose(np.mean(v0), 0, atol=1e-2)
+    assert np.allclose(np.std(v0), 1/np.sqrt(3), atol=1e-3)
```

Comment:
Nit: (This way we can avoid using the two for loops in the list comprehension)

```suggestion
	v0 = np.concatenate(v0s)
	assert np.all(v0 <= 1)
    assert np.all(v0 >= -1)
    
    assert_allclose(np.mean(v0), 0, atol=1e-2)
    assert_allclose(np.std(v0), 1/np.sqrt(3), atol=1e-3)
```

Where `from numpy.testing import assert_allclose`